### DISCUSSION TITLE
how to upgrade Mermaid module version

### DISCUSSION THREAD

**Discussion by wellsleep** [requarks/wiki#4392] [Help / Questions]
Hi guys,

From `package.json` I'm able to find that the Mermaid module is at 8.8.2. 
Is it possible to upgrade it to a new version, like 8.12.x? 
Because the new features in latest Mermaid is very useful to my case (e.g. direction in subgraph).

Thanks in advance,

my env:
- Ubuntu 20.04 with dockered wikijs at 2.5.201

**Comment by elmorejd** [comment]
I too would like to see this get updated to the latest version. There are quite a lot of nice features added since the included version.
